index.js

```diff
@@ -25,12 +25,39 @@ const panel = require('sdk/panel').Panel({
   }
 });
 
+// Init the panel's location state
+panel.coords = { bottomOffset: -10, leftOffset: 10 };
+
 getActiveView(panel).setAttribute('noautohide', true);
 
-// check for linux here
-// if (system.platform !== 'linux') {
-//   makePanelDraggable(panel);
-// }
+// Draggability seems to work for windows and mac, but not linux.
+if (system.platform === 'winnt' || system.platform === 'darwin') {
+  makePanelDraggable(panel);
+}
+
+function adjustHeight(newHeight) {
+  const xulPanel = getActiveView(panel);
+  const stack = xulPanel.getElementsByTagName('stack')[0];
+  const frame = xulPanel.getElementsByTagName('iframe')[0];
+
+  frame.setAttribute('height', newHeight);
+  stack.setAttribute('height', newHeight);
```

Comment:
Oh, never mind. The stack is only created if you makePanelDraggable.

I can just loop over the child elements when I adjustHeight